lib/src/widgets/chat/ds_interactive_voice_call_message_bubble.widget.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+
+import '../../enums/ds_align.enum.dart';
+import '../../enums/ds_border_radius.enum.dart';
+import '../../models/ds_message_bubble_style.model.dart';
+import '../../models/interactive_message/ds_interactive_message.model.dart';
+import '../../themes/colors/ds_colors.theme.dart';
+import '../../themes/icons/ds_icons.dart';
+import '../texts/ds_body_text.widget.dart';
+import '../utils/ds_divider.widget.dart';
+import 'ds_message_bubble.widget.dart';
+
+class DSInteractiveVoiceCallMessageBubble extends StatelessWidget {
+  final DSInteractiveMessage content;
+  final DSAlign align;
+  final List<DSBorderRadius> borderRadius;
+  final DSMessageBubbleStyle style;
+
+  late final bool _isLightBubbleBackground;
+
+  DSInteractiveVoiceCallMessageBubble({
+    super.key,
+    required this.content,
+    required this.align,
+    this.borderRadius = const [DSBorderRadius.all],
+    DSMessageBubbleStyle? style,
+  }) : style = style ?? DSMessageBubbleStyle() {
+    _initProperties();
+  }
+
+  void _initProperties() {
+    _isLightBubbleBackground = style.isLightBubbleBackground(align);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return DSMessageBubble(
+      align: align,
+      style: style,
+      borderRadius: borderRadius,
+      child: Column(
+        children: [
+          DSBodyText(
+            content.body?.text,
+            overflow: TextOverflow.visible,
+            color: _isLightBubbleBackground
```

Comment:
To avoid repeating this check, you could store it's value in a variable and use it instead.